tests/test_SE_combine_methods.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import pandas as pd
+import pytest
+from summarizedexperiment.SummarizedExperiment import SummarizedExperiment
+
+__author__ = "keviny2"
+__copyright__ = "keviny2"
+__license__ = "MIT"
+
+
+rowData1 = pd.DataFrame(
+    {"meta1": ["val_1", "val_2", "val_3"], "meta2": ["elem_1", "elem_2", "elem_3"]},
+    index=["HER2", "BRCA1", "TPFK"],
+)
+colData1 = pd.DataFrame(
+    {"meta1": ["val_1", "val_2", "val_3"], "meta2": ["dat_1", "dat_2", "dat_3"]},
+    index=["cell_1", "cell_2", "cell_3"],
+)
+se1 = SummarizedExperiment(
+    assays={"counts": np.random.poisson(lam=5, size=(3, 3))},
+    rowData=rowData1,
+    colData=colData1,
+    metadata={"query": "SOME awesome QUERY"},
+)
+
+rowData2 = pd.DataFrame(
+    {"meta1": ["val_1", "val_2", "val_3"], "meta2": ["elem_1", "elem_2", "elem_3"]},
+    index=["MYC", "BRCA2", "GSS"],
+)
+colData2 = pd.DataFrame(
+    {"meta1": ["val_1", "val_2", "val_3"], "meta2": ["dat_1", "dat_2", "dat_3"]},
+    index=["cell_4", "cell_5", "cell_6"],
+)
+se2 = SummarizedExperiment(
+    assays={"counts": np.random.poisson(lam=5, size=(3, 3))},
+    rowData=rowData2,
+    colData=colData2,
+    metadata={"gene_list": "MYC, BRCA2, GSS"},
+)
+
+rowData3 = pd.DataFrame(
+    {"meta1": ["val_1", "val_2", "val_3"], "meta2": ["elem_1", "elem_2", "elem_3"]},
+    index=["MYC", "BRCA2", "GSS"],
+)
+colData3 = pd.DataFrame(
+    {"meta1": ["val_1", "val_2", "val_3"], "meta2": ["dat_1", "dat_2", "dat_3"]},
+    index=["cell_7", "cell_8", "cell_9"],
+)
+se3 = SummarizedExperiment(
+    assays={"counts": np.random.poisson(lam=5, size=(3, 3))},
+    rowData=rowData3,
+    colData=colData3,
+)
+
+rowData4 = pd.DataFrame(
+    {
+        "meta1": ["val_1", "val_2", "val_3", "val_4"],
+        "meta2": ["elem_1", "elem_2", "elem_3", "elem_4"],
+    },
+    index=["MYC", "BRCA2", "GSS", "TPFK"],
+)
+colData4 = pd.DataFrame(
+    {
+        "meta1": ["val_1", "val_2", "val_3"],
+        "meta2": ["dat_1", "dat_2", "dat_3"],
+    },
+    index=["cell_7", "cell_8", "cell_9"],
+)
+se4 = SummarizedExperiment(
+    assays={"counts": np.random.poisson(lam=5, size=(4, 3))},
+    rowData=rowData4,
+    colData=colData4,
+)
+
+rowData5 = pd.DataFrame(
+    {
+        "meta1": ["val_1", "val_2", "val_3", "val_4"],
+        "meta2": ["elem_1", "elem_2", "elem_3", "elem_4"],
```

Comment:
tests are missing a few scenarios

- [ ] all row datas seem to have `meta1` & `meta2` as column names, should use a combination where one column exists in a few SE's and some don't
- [ ] same applies to col data
- [ ] all SE's have `counts`, should have a combination of assay names across SE's